src/testhost/testhost.csproj

```diff
@@ -33,6 +33,11 @@
       <Version>4.3.0</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
```

Comment:
Why is this required? Just curious since we had faced several issues with TypeConverter earlier, and not sure if TypeConverter 4.3.0 is compatible with UAP10.0?